### ISSUE TITLE
map transparency functionality has not been implemented for point maps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#209]
```
 I have a basemap (Nokia) and the transparency dialog comes
up, but it doesn’t seem to do anything. 
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 5 Aug 2015 at 11:11

**Comment by lixun910** [comment]
This fix will be applied in 1.7.47

**Comment by jkoschinsky** [comment]
Transparency slider now works with points and polygons in GeoDa 1.7.63.

JK: verify in Windows

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.89 (Win 7 Professional (64-bit)).
